www/locale/fi/LC_MESSAGES/django.po

```diff
@@ -417,11 +419,11 @@ msgstr ""
 #: www/templates/www/user.html:82
 #, python-format
 msgid ""
-"Access to space should be paid every month with reference number %(ref)s\n"
-"to bank account %(bank_iban)s. Service price is %(cost)s."
+"This service can be paid with reference number %(ref)s\n"
+"                to bank account %(bank_iban)s. Service price is %(cost)s."
 msgstr ""
-"Tilankäyttöoikeus tulee maksaa kuukausittain viitenumerolla %(ref)s tilille %(bank_iban)s. "
-"Palvelun hinta on %(cost)s."
+"Tilankäyttöoikeus tulee maksaa kuukausittain viitenumerolla %(ref)s\n"
```

Comment:
```suggestion
"Tämä palvelu tulee maksaa kuukausittain viitenumerolla %(ref)s\n"
```

The service can be something different thank tilankäyttöoikeus.

Reply:
hmm, even "palvelu" is a bit strange here.. maybe this should contain the service name?

Reply:
Maybe, but needs also mod to english, so let's do that separately :)